content/doc/book/installing/_docker.adoc

```diff
@@ -21,28 +21,33 @@ _run-jenkins-in-docker.adoc).
 [source,bash]
 ----
 docker run \
-  -u root \
-  --rm \  # <1>
-  -d \ # <2>
-  -p 8080:8080 \ # <3>
-  -p 50000:50000 \ # <4>
-  -v jenkins-data:/var/jenkins_home \ # <5>
-  -v /var/run/docker.sock:/var/run/docker.sock \ # <6>
-  jenkinsci/blueocean # <7>
+  -u root \ # <1>
+  --rm \ # <2>
+  -d \ # <3>
+  -p 8080:8080 \ # <4>
+  -p 50000:50000 \ # <5>
+  -v jenkins-data:/var/jenkins_home \ # <6>
+  -v /var/run/docker.sock:/var/run/docker.sock \ # <7>
+  jenkinsci/blueocean # <8>
 ----
-<1> ( _Optional_ ) Automatically removes the Docker container (which is the
+<1> ( _Optional_ ) Since the default `jenkins` user doesn't have access to
+`/var/run/docker.sock`, you need to run Jenkins as `root` to allow Jenkins to
+spawn docker containers to execute stages in your pipelines. This only affects
+executors on the master Jenkins node, so if you are planning to execute your
+pipelines on slaves, this is not necessary.
```

Comment:
Our convention is to say "agents" rather than "slaves."

Reply:
```suggestion
pipelines on agents, this is not necessary.
```